views/forms/BarChartFields.jsx

```diff
@@ -118,6 +145,29 @@ const BarChartFields = ({
               <Delete className={classes.icon} />
             </Button>
           </div>
+          <div key={idx} className={classes.formLabelRow}>
+            <Typography variant='h6' color="textSecondary">
+              Target Values
+            </Typography>
+          </div>
+          {field.targetValues.length > 0 && field.targetValues.map((target, tidx) => (
```

Comment:
why do we need to check `length` here?

Reply:
[d28f2d6](https://github.com/AMP-SCZ/dpdash/pull/110/commits/d28f2d6b1be0f5c994babe2ddbf5581cbeed790c)

Reply:
[looks like we're still checking length](https://github.com/AMP-SCZ/dpdash/pull/110/commits/d28f2d6b1be0f5c994babe2ddbf5581cbeed790c#diff-fbb0f165f85d2c475bce82a02e022f5ab33830668e956de1139b5eef41a55a13R148) -- If `targetValues` is an empty array, and we map over it returning JSX, [react won't render anything there](https://codesandbox.io/s/elastic-cannon-t7hw48?file=/src/App.tsx). It doesn't hurt to check the length, but it isn't necessary to do so 🙂

Reply:
If I don't check for length then I get this zero on the view.
<img width="770" alt="Screen Shot 2022-07-25 at 8 53 59 PM" src="https://user-images.githubusercontent.com/19805355/181016593-3637b0b3-f8e2-4bf5-946e-12b473e15a7b.png">

Reply:
huh, quick huddle?